### ISSUE TITLE
heading elements not in sequential order

### ISSUE THREAD

**Issue by Deep-Codes** [leerob/leerob.io#299]
You seem to be missing the perfect lighthouse score because of accessibility.

<img width="597" alt="Screenshot 2021-03-29 at 7 46 54 PM" src="https://user-images.githubusercontent.com/61158210/112851118-4b881300-90c8-11eb-81f5-57d2c50d002c.png">

Code causing the Problem:

```jsx
 <h5 className="text-lg md:text-xl font-bold text-gray-900 dark:text-gray-100">
    Subscribe to the newsletter
 </h5>
```

We can convert this to the `h4` tag and solve the issue for the index page but it causes a problem in blogs/ page.

But Since your using Tailwind CSS you can replace it with some other tag like `p` and retain the styles regardless.

Not a Big Problem tbh but Pointed it out Since huge fan of this Repo 💙.
